app/views/api/vacancies/index.json.jbuilder

```diff
@@ -1,3 +1,19 @@
-json.vacancies @vacancies.decorated_collection do |vacancy|
+json.info do
+  json.title "GOV UK - #{I18n.t('app.title')}"
+  json.description I18n.t('app.description')
+  json.termsOfService I18n.t('service.url')
```

Comment:
Can we use a rails url method to return `https://teaching-jobs.service.gov.uk/pages/terms-and-conditions` here?